prow/plugins/cat/cat.go

```diff
@@ -140,7 +141,7 @@ func (c *realClowder) URL(category string, movieCat bool) string {
 func (c *realClowder) readCat(category string, movieCat bool) (string, error) {
 	cats := make([]catResult, 0)
 	uri := c.URL(category, movieCat)
-	if strings.TrimSpace(category) == "no" {
+	if grumpyKeywords.FindStringSubmatch(category) != nil {
```

Comment:
Since we don't care about the content of the match (in this case whether or not the matched category was `no` or `grumpy`) it is sufficient to use [`MatchString`](https://golang.org/pkg/regexp/#Regexp.MatchString) which returns a `bool` instead of a `[]string`.